website/docs/r/pki_secret_backend_config_est.html.md

```diff
@@ -0,0 +1,109 @@
+---
+layout: "vault"
+page_title: "Vault: vault_pki_secret_backend_config_est resource"
+sidebar_current: "docs-vault-resource-pki-secret-backend-config-est"
+description: |-
+  Sets the EST configuration on a PKI Secret Backend for Vault.
+---
+
+# vault\_pki\_secret\_backend\_config\_est
+
+Allows setting the EST configuration on a PKI Secret Backend
+
+## Example Usage
+
+```hcl
+resource "vault_mount" "pki" {
+	path        = "pki-root"
+	type        = "pki"
+    description = "PKI secret engine mount"
```

Comment:
nit: indentation
```suggestion
	description = "PKI secret engine mount"
```

Reply:
Fixed in https://github.com/hashicorp/terraform-provider-vault/pull/2246/commits/5f5961a4a31806fc4509223e1234830c7a669590

I actually fixed the lines above to match up with the indentation across the rest of the example.